lib/main.dart

```diff
@@ -42,8 +42,8 @@ class _RandomWordsState extends State<RandomWords> {
         style: _biggerFont,
       ),
       trailing: Icon(
-        alreadySaved ? Icons.favorite : Icons.favorite_border,
-        color: alreadySaved ? Colors.red : null,
+        alreadySaved ? Icons.done : Icons.attach_money,
+        color: alreadySaved ? null : Colors.green,
```

Comment:
I'm not sure, but if the user taps to save some startup name, the icon will disappear, am I right?
If so, I would instead give the user a possibility to discard their choice, for example.

Reply:
I don't recommend using `null` for colors ever :)